src/components/Case-study-banner/index.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
```

Comment:
I assume in this file we were not using useEffect. @Ykumar1415 ?

Reply:
yes @samyakshah3008  that's why i removed it.

Reply:
Sounds cool.